rllib/algorithms/appo/tf/appo_tf_policy_rlm.py

```diff
@@ -62,6 +61,9 @@ def __init__(self, observation_space, action_space, config):
         KLCoeffMixin.__init__(self, config)
         GradStatsMixin.__init__(self)
         EagerTFPolicyV2.__init__(self, observation_space, action_space, config)
+        self.target_model = self.make_rl_module()
+        self.target_model.set_weights(self.model.get_weights())
+        # construct the target model and make its weights the same as the model
```

Comment:
comment should come before the implementation :)

Reply:
done